easybuild/easyconfigs/g/GDAL/GDAL-1.10.1-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '1.10.1'
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
```

Comment:
the `%(version)s` part didn't used to be there, this should also be changed in other GDAL easyconfigs (separate PR pls)?